projects/1.16/assets/relics-of-gaming/rogblades/lang/zh_cn.json

```
@@ -0,0 +1,43 @@
+{
+    "itemGroup.rog_mod":"游戏遗物",
+ 
+   "rogblades.name": "游戏遗物",
+   "rogblades.desc": "欢迎来到游戏遗物！",
+ 
+   "item.slashblade.ex.beastlord": "遗物「兽王」",

```

```suggestion
   "item.slashblade.ex.beastlord": "百兽剑王",
```
库内的拔刀剑2完全遵循原文翻译，不再延续拔刀剑1的命名模式
英语里的`--`等同于中文里的`「」`
刀、刀组件的格式问题请在所有翻译文件中查明并修改

此外武器neta了尼尔系列的百兽剑王：https://www.bilibili.com/read/cv11251513/